assist/bin/rack/write_ontology.pl

```diff
@@ -153,12 +161,12 @@
     string_lower(Namespace, Module),
     writeln(Handle, '% THIS FILE WAS AUTOMATICALLY GENERATED, SEE README'),
     nl(Handle),
-    atomic_list_concat([':- module(', Module, ','], FirstLine),
-    writeln(Handle, FirstLine),
-    writeln(Handle, '          ['),
-    write_exports_for_things(Handle, Things),
-    write_exports_for_properties(Handle, Properties),
-    writeln(Handle, '          ]).'),
+    writeln(Handle, ':- module('),
+    atomic_list_concat(['    ', Module, ','], ModuleLine),
```

Comment:
The indentation here is still hard-coded, relative to what `export_indentation` will provide; it's not terribly important, but at some point we might want to pass an indentiation level that gets incremented and decremented.  Just a comment though, probably not worth spending time on right now.

Reply:
That's actually fairly easy to do in a nicer way, fixed it in 8b9726f

Reply:
Nice!